src/triage/component/architect/feature_query_runners.py

```diff
@@ -0,0 +1,141 @@
+import logging
+
+import sqlparse
+
+from triage.database_reflection import table_exists
+
+
+def run_statements(statement_list, db_engine):
+    with db_engine.begin() as conn:
+        for statement in statement_list:
+            logging.debug("Executing feature generation query: %s", statement)
+            conn.execute(statement)
+
+
+def process_table_task(task, db_engine):
+    run_statements(task.get("prepare", []), db_engine)
+    run_statements(task.get("inserts", []), db_engine)
+    run_statements(task.get("finalize", []), db_engine)
+
+
+def process_table_tasks(table_tasks, db_engine, verbose=False):
+    for task, task_num in enumerate(table_tasks, 1):
+        if verbose:
+            log_verbose_task_info(task, task_num)
+        process_table_task(task, db_engine)
+
+
+def needs_features(feature_block, db_engine):
```

Comment:
could/should `FeatureBlock.needs_features` use this, rather than copying it, or something?